docs/debug.rst

```diff
@@ -111,84 +65,74 @@ And the export config:
     +----------+-------+----------+------+-----------+--------------+
     1 row in set (0.00 sec)
 
-If you are missing either of these entries, you might have setup your
-database as part of a development setup and used the `./dev test`
-script. This sets up the database for testing needs, but doesn't
-configure the export config required for running a real application setup.
-
-One way to correct this is to downgrade and re-upgrade the database
-schema using alembic. Using the appropriate docker invocation, you
-can run:
-
-.. code-block:: bash
-
-    <...docker or dev script...> alembic downgrade base
-    <...docker or dev script...> alembic upgrade head
-
-This tears down the database schema to an empty state and upgrades it
-again. As part of the upgrade, the API key and export config are added.
+If you are missing either of these entries, then it's likely you need to set up
+API keys and export configuration.
 
 
 Connections
 -----------
 
-Docker containers typically run on their own IP sub-network and
-connecting from inside an application container to a different
-machine can be tricky, especially if those other machines are also
-docker containers.
+Check if the web service and celery containers can connect to the MySQL
+database and Redis datastore.
 
-Best to check if the containers can actually reach the MySQL database
-and Redis datastore.
+Follow the instructions in the `Runtime Checks` part of the :ref:`the deploy
+docs <deploy>`. Make sure to call the ``/__heartbeat__`` HTTP endpoint on the
+web application.
 
-The easiest way is to follow the instructions in the `Runtime Checks`
-part of the :ref:`the deploy docs <deploy>`. Especially calling the
-`/__heartbeat__` HTTP endpoint on the web application.
+Another way to check connections is to start a container and try to connect to
+the two external connections from inside it.
 
-Another way is to start a container and try to connect to the two
-external connections from inside it:
+In a local development environment, you can do this:
 
 .. code-block:: bash
 
-    <...docker or dev script...> shell
+   make shell
+
+In a server environment, you need to run the container with configuration in
+the environment.
 
-And inside the container:
+Once inside the container, you can do this:
 
 .. code-block:: bash
 
-    redis-cli -h $REDIS_HOST
+    $ redis-cli -u $REDIS_URI
```

Comment:
Nice!

Reply:
I'm pretty happy with this, but tossing around creating a make rule for it.